share/spice/book/book.js

```diff
@@ -115,8 +115,9 @@
             },
 
             templates: {
-                group: 'info',
+                group: 'base',
                 options: {
+                    moreAt: true,
                     description: true,
                     content: Spice.book.book
```

Comment:
This template should really be renamed to "content.handlebars" -- can fix or we can make a task, its nbd.